src/Tools/Github/GitMergeBot/Program.cs

```diff
@@ -83,7 +83,14 @@ private Program(Options options)
             }
 
             var newBranchName = await MakePrBranch(_options.SourceUser, _options.RepoName, remoteIntoBranch, newBranchPrefix);
-            await SubmitPullRequest(newBranchName);
+            var pullRequest = await SubmitPullRequest(newBranchName);
+
+            // pullRequest could be null if we are running in debug mode.
+            if (pullRequest != null)
+            {
+                await Task.Delay(2000);
+                await _client.Issue.Comment.Create(_options.DestinationUser, _options.RepoName, pullRequest.Number, "@dotnet-bot test vsi please");
```

Comment:
Only issue this comment if there were no conflicts, e.g., the number of unmerged files was zero.